tests/test_gitautosync.py

```diff
@@ -3,10 +3,10 @@
 import subprocess
 import random
 import string
-from gitautosync.gitautosync import GitAutoSync
+from gitautosync import GitAutoSync
 
 
-class TestGitAutoSync(object):
+class TestGitAutoSync:
```

Comment:
Should we write more unit tests for some of the new code added, like `execute_cmd` and the new handlers? Or do you think that's not necessary?